api/docker/caddy/Caddyfile

```diff
@@ -20,7 +20,7 @@
 		}
 	}
 
-	handle_path /print* {
+	handle /print* {
```

Comment:
the difference between handle and handle_path is that handle_path matches the url und removes the prefix,
while handle does not remove the prefix.
[GRGR.pdf](https://github.com/ecamp/ecamp3/files/11786191/GRGR.pdf)
(output of the e2e tests)

If you change the routing, you also have to change it in the ingress controller,
in the kubernetes deployment, caddy does not handle the url rewriting in the kubernetes deployment.
https://github.com/ecamp/ecamp3/blob/devel/.helm/ecamp3/templates/print_ingress.yaml#L14,L15
and https://github.com/ecamp/ecamp3/blob/devel/.helm/ecamp3/templates/print_ingress.yaml#L28

what is the problem you want to solve?

It seems to be possible to load assets with the current routing config (https://github.com/ecamp/ecamp3/blob/8ada8c9f2cc5ad1af652752dbd6aa6
1de7daf3b5/print/nuxt.config.js#L29)

Reply:
Assets can't be loaded, as it tries to load them from `/_nuxt/`, without a baseURL this will result in `localhost:3000/_nuxt/` which won't resolve. If I set the baseURL to `/print` without changing caddy, then it gets caught in an endless redirect loop. 

These stylesheets get embedded as `<style>` tags, and Open Sans is currently imported from Google Fonts. That's why it does not fail currently.

Could you help me adjusting the ingress controller?

Reply:
Nope, there are also `/_loading/`, which paths can't be adjusted.